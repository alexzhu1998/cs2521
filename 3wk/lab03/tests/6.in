2048
512
1792
1024
8198
2560
4096
1536
3072
1280
3584
1408
3328
1344
3456
1376
3392
1360
1352
1348
1350
1351
1392
1400
1404
1402
1401
3360
3344
3336
3340
3342
3424
3440
3448
3444
3442
