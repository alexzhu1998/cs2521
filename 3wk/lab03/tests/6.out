Tree:
                  2048
                   / \
                  /   \
                 /     \
                /       \
               /         \
              /           \
             /             \
            /               \
           /                 \
          /                   \
         /                     \
        /                       \
      512                      8198
        \                       /
       1792                   2560
        /                       \
      1024                     4096
        \                       /
       1536                   3072
        /                       \
      1280                     3584
        \                       /
       1408                   3328
        /                       \
      1344                     3456
        \                       /
       1376                   3392
        / \                   / \
       /   \                 /   \
      /     \               /     \
    1360   1392           3360   3424
    /         \           /         \
  1352       1400       3344       3440
  /             \       /             \
1348           1404   3336           3448
  \             /       \             /
 1350         1402     3340         3444
    \         /           \         /
   1351     1401         3342     3442

#nodes = 37,  #leaves = 4
Infix  : 512 1024 1280 1344 1348 1350 1351 1352 1360 1376 1392 1400 1401 1402 1404 1408 1536 1792 2048 2560 3072 3328 3336 3340 3342 3344 3360 3392 3424 3440 3442 3444 3448 3456 3584 4096 8198 
Prefix : 2048 512 1792 1024 1536 1280 1408 1344 1376 1360 1352 1348 1350 1351 1392 1400 1404 1402 1401 8198 2560 4096 3072 3584 3328 3456 3392 3360 3344 3336 3340 3342 3424 3440 3448 3444 3442 
Postfix: 1351 1350 1348 1352 1360 1401 1402 1404 1400 1392 1376 1344 1408 1280 1536 1024 1792 512 3342 3340 3336 3344 3360 3442 3444 3448 3440 3424 3392 3456 3328 3584 3072 4096 2560 8198 2048 
ByLevel: 2048 512 8198 1792 2560 1024 4096 1536 3072 1280 3584 1408 3328 1344 3456 1376 3392 1360 1392 3360 3424 1352 1400 3344 3440 1348 1404 3336 3448 1350 1402 3340 3444 1351 1401 3342 3442 
